src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs

```diff
@@ -2,12 +2,16 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 // This program uses code hyperlinks available as part of the HyperAddin Visual Studio plug-in.
 // It is available from http://www.codeplex.com/hyperAddin 
+#if !PLATFORM_UNIX
+
 #define FEATURE_MANAGED_ETW
```

Comment:
I think that just making that change is insufficient.  Right now, the only place FEATURE_ACTIVITYSAMPLING appears to be defined is in this section of EventSource.cs; if I change the #if !PLATFORM_UNIX to instead be #if !INSIDE_CLR, then I believe FEATURE_ACTIVITYSAMPLING will go from being defined to not being defined in mscorlib on Windows.  Given that addressing that would then lead to some more cleanup and refactoring, and since we don't currently have full test validation of this on GitHub, I'm going to leave it as-is for now.  It sounds like there are more changes coming to this file and area in the near future, anyway, and this is all going to get additional treatment as we figure out what to hook EventSource up to by default on Unix.